### DISCUSSION TITLE
How to setup Vercel log drain for Google Cloud Logging (formerly StackDriver)?

### DISCUSSION THREAD

**Discussion by GorvGoyl** [vercel/vercel#6597] [Help]
[Google Cloud Logging](https://cloud.google.com/logging/) integration is not available in Vercel marketplace (https://vercel.com/integrations#logging) and I couldn't find any resource on setting it up on the web either. Is it possible to do so? I'd be glad if someone could share some resources.

**Comment by leerob** [comment]
Hey! I will look into this and let you know what I find. Sorry there isn't any documentation on this.

**Reply by GorvGoyl** [reply]
hey Lee, did you get chance to look into it?

Google cloud logging is quite powerful as compared to mentioned logging solutions in vercel marketplace. It also has a very generous free tier, better UI, and a dedicated mobile app to get error alerts etc.

**Reply by jthompson22** [reply]
Is there an update on this?

**Comment by praveentcom** [comment]
It's been a while since this thread initiated. Quite surprising why it's not yet added to Vercel. GCP Cloud Logging is needed and is a go-to source for many.

**Comment by bieschke** [comment]
+1

**Comment by AlekseyMko** [comment]
+1

**Comment by JakubKontra** [comment]
+1

**Comment by sriramsundhar** [comment]
+1

**Reply by steebchen** [reply]
Please upvote or thumbs-up the original comment, and please don't write `+1` comments as it notifies everyone subscribed to this thread without any new information. Thanks.
